metrics/lib/common.bash

```diff
@@ -42,6 +42,49 @@ function save_results(){
 	bash $LIB_DIR/send_results.sh -n "$1" -a "$2" -r "$3" -u "$4"
 }
 
+# This function checks existence of commands.
+# They can be received standalone standalone or as an array, e.g.
+#
+# cmds=(“cmd1” “cmd2”)
+# check_cmds "${cmd[@]}"
+function check_cmds()
+{
+	req_tools="$@"
+	for cmd in ${req_tools[*]}; do
+		command -v "$cmd" &> /dev/null
+		res=$?
```

Comment:
Hi @MarioCarrilloA I tried a test of this on my Ubuntu machine, and I got silent failure I believe as the script invoking this (fio_tests.sh in this case) has `-e` set, so quits at the 'command' failure instead of calling the `die` command.
I added a simple `set +e` inside check_cms() to verify this, and that did seen to be the issue. I don't know if that is the best solution though. /cc @bwarden both for any input on the shell magic here, and also as I think @bwarden should be aware of the functions you are adding here so we can review/share/re-use across all the metrics tests.